poll/obs/get.js

```diff
@@ -0,0 +1,189 @@
+const pull = require('pull-stream')
+const PullNotify = require('pull-notify')
+const sort = require('ssb-sort')
+const { Struct, Value, Array: MutantArray, computed, resolve } = require('mutant')
+const getContent = require('ssb-msg-content')
+const { isPoll, isPosition, isChooseOnePoll, isPollUpdate, isChooseOnePosition } = require('ssb-poll-schema')
+isPoll.chooseOne = isChooseOnePoll
+isPosition.chooseOne = isChooseOnePosition
+const buildResults = require('../../results/sync/buildResults')
+const { CHOOSE_ONE } = require('../../types')
+
```

Comment:
can rm these notes

Reply:
Yes. Intentionally left them there so you understood _why_ it looked like it did.